packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1068,9 +1134,42 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return this._summarizer;
     }
 
-    private get summariesDisabled(): boolean {
-        return this.runtimeOptions.summaryOptions.disableSummaries === true ||
-            this.runtimeOptions.summaryOptions.summaryConfigOverrides?.disableSummaries === true;
+    public get summariesDisabled(): boolean {
+        if (this.runtimeOptions.summaryOptions.disableSummaries === true) {
```

Comment:
Can you please add a comment saying this is for back-compat? That will make it easier to understand.

Reply:
+1: Please follow add comment in this format:
// back-compat: Added in [Version added].